<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CsvViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="openButton"/>
    <addaction name="settingsButton"/>
    <addaction name="exitButton"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="aboutButton"/>
   </widget>
   <widget class="QMenu" name="actionMenu">
    <property name="title">
     <string>Действия</string>
    </property>
    <addaction name="filterButton"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="actionMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openButton">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="settingsButton">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="exitButton">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="aboutButton">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="filterButton">
   <property name="text">
    <string>Фильтр</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
